### PR TITLE
add a country.html

### PR DESCRIPTION
**Pull request by ferolinojasmin**
added a country.html

### CODE REVIEW

File: ferolino_jasmin/country.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    <input type="text" id="countryInput" placeholder="Enter a country">
+    <button onclick="searchCountry()">Search</button>
+
+    <div id="countryDetails">
+    </div>
+
+    <div id="countriesInRegion">
+    </div>
+
+    <script src="countries.js"></script>
```
Comment (nabayra-james) [inline ferolino_jasmin/country.html]: This is not the name of your js file. javascript.js is the appropriate file name.

File: ferolino_jasmin/javascript.js [hunk 2]
```diff
@@ -79,4 +79,49 @@ function renderComments() {
 }
 
 sortDropdown.addEventListener("change", renderComments);
-renderComments()
\ No newline at end of file
+renderComments()
+
+async function searchCountry() {
+  const countryInput = document.getElementById('countryInput').value;
```
Comment (nabayra-james) [inline ferolino_jasmin/javascript.js]: ### IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies to lines 95 and 113 in your JS file.

File: ferolino_jasmin/country.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<body>
+    <h1>Countries of the World</h1>
+    <input type="text" id="countryInput" placeholder="Enter a country">
```
Comment (nabayra-james) [inline ferolino_jasmin/country.html]: ### IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies to lines 6, and 9.

**Review by nabayra-james — CHANGES_REQUESTED** [review]

File: ferolino_jasmin/country.html [hunk 4]
```diff
@@ -8,15 +8,15 @@
 </head>
 <body>
     <h1>Countries of the World</h1>
-    <input type="text" id="countryInput" placeholder="Enter a country">
+    <input type="text" id="country_input" placeholder="Enter a country">
     <button onclick="searchCountry()">Search</button>
 
-    <div id="countryDetails">
+    <div id="countrydetails">
```
Comment (nabayra-james) [inline ferolino_jasmin/country.html]: ### IDs should be written in snake case. All lower case separated by underscore.

This the appropriate id: country_details

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

This also applies in line 17

File: ferolino_jasmin/country.html [hunk 5]
```diff
@@ -8,15 +8,15 @@
...
-    <div id="countryDetails">
+    <div id="countrydetails">
     </div>
 
-    <div id="countriesInRegion">
+    <div id="countrieinregion">
     </div>
 
-    <script src="countries.js"></script>
+    <script src=" javascript.js"></script>
```
Comment (nabayra-james) [inline ferolino_jasmin/country.html]: ### Remove unnecessary/excessive lines and spacing.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d18931cb7_3_0

No space in the script tag for the file name of your js.

**Review by nabayra-james — DISMISSED** [review]

**Review by Japitana-JohnWalter — APPROVED** [review]

**Pull request merged**